virtual_ship/spacetime.py

```diff
@@ -7,7 +7,6 @@
 
 
 @dataclass
-# TODO I take suggestions for a better name
```

Comment:
I think LocationTime would already be better (less StarTrek ;-) )

Reply:
let me do this in another pr